testsuite/podman_runner/08_manager_setup.sh

```diff
@@ -3,8 +3,11 @@ set -xe
 
 src_dir=$(cd $(dirname "$0")/../.. && pwd -P)
 
+# mgrctl should not be installed in this container
+sudo -i podman exec server bash -c "rm /usr/bin/mgrctl"
```

Comment:
Why exactly it affects us?

Reply:
I can't recall which test was, but there is a test that tries to run a command inside the server container with "mgrctl exec", because it thinks it is running in the host.

Reply:
This looks hacky! I'ld like to know where we are checking for `mgrctl` inside the server container...

Reply:
IIRC, was extracting log files. Instead of using scp, it was using "mgrctl exec". Note here there is no "host", the "host" and the "container" are the same thing.

Reply:
checking here https://github.com/uyuni-project/uyuni/pull/9572

Reply:
I would rather hack `has_mgrctl` detection in the ruby code to ignore it if running inside a container than removing the file here

Reply:
just trying to run core tests fail

```
/usr/bin/ruby.ruby3.3 -S bundle exec cucumber --profile default -f html -o results/output_20241216154624-github_validation_core.html -f json -o results/output_20241216154624-github_validation_core.json -f junit -o results_junit -f pretty -r features --fail-fast features/github_validation/core/first_settings.feature features/core/srv_user_preferences.feature features/reposync/srv_create_fake_channels.feature features/reposync/srv_create_fake_repositories.feature features/reposync/srv_sync_fake_channels.feature features/reposync/srv_create_activationkey.feature features/core/srv_osimage.feature features/core/srv_docker.feature
Proxy IP address or domain name variable empty
PXE boot MAC address variable empty
KVM server minion IP address or domain name variable empty
FAIL: mgrctl exec -i 'rm -f /etc/motd && touch /etc/motd' returned status code = 1.
Output:
 (ScriptError)
/testsuite/features/support/remote_node.rb:119:in `run_local'
/testsuite/features/support/remote_node.rb:103:in `run'
/testsuite/features/support/remote_node.rb:40:in `initialize'
/testsuite/features/support/remote_nodes_env.rb:37:in `new'
/testsuite/features/support/remote_nodes_env.rb:37:in `get_target'
/testsuite/features/support/commonlib.rb:630:in `new_api_client'
/testsuite/features/support/env.rb:122:in `<top (required)>'
/usr/lib64/ruby/3.3.0/bundled_gems.rb:69:in `require'
/usr/lib64/ruby/3.3.0/bundled_gems.rb:69:in `block (2 levels) in replace_require'
/usr/lib64/ruby/gems/3.3.0/gems/cucumber-9.2.0/lib/cucumber/glue/registry_and_more.rb:124:in `load_code_file'
/usr/lib64/ruby/gems/3.3.0/gems/cucumber-9.2.0/lib/cucumber/runtime/support_code.rb:145:in `load_file'
/usr/lib64/ruby/gems/3.3.0/gems/cucumber-9.2.0/lib/cucumber/runtime/support_code.rb:82:in `block in load_files!'
/usr/lib64/ruby/gems/3.3.0/gems/cucumber-9.2.0/lib/cucumber/runtime/support_code.rb:81:in `each'
/usr/lib64/ruby/gems/3.3.0/gems/cucumber-9.2.0/lib/cucumber/runtime/support_code.rb:81:in `load_files!'
/usr/lib64/ruby/gems/3.3.0/gems/cucumber-9.2.0/lib/cucumber/runtime.rb:274:in `load_step_definitions'
/usr/lib64/ruby/gems/3.3.0/gems/cucumber-9.2.0/lib/cucumber/runtime.rb:74:in `run!'
/usr/lib64/ruby/gems/3.3.0/gems/cucumber-9.2.0/lib/cucumber/cli/main.rb:29:in `execute!'
/usr/lib64/ruby/gems/3.3.0/gems/cucumber-9.2.0/bin/cucumber:9:in `<top (required)>'
/usr/bin/cucumber:25:in `load'
/usr/bin/cucumber:25:in `<main>'
Using Ruby version: 3.3.6
Capybara APP Host: https://server:8888
Initializing a remote node for 'server'.
```
Can't initialize the server node

Reply:
I see, ok, well, in theory Marina will fix that issue about having the mgrctl inside the container and that problem should gone.
But I will see if we can do something around how we detect where we are.

Out-of-scope:
At some point, long term, I envision that we will have environments that will not have any kind of mgr-tools packages, going closer to a proper clould friendly product, so, it will be good to don't rely that much on that mgrctl to identify where we are.

Reply:
next container release won't have the mgrctl package ... we could wait for that release...